### PR TITLE
Publishes ONRR job listing

### PR DESCRIPTION
**Pull request by elainekamlley**
Fixes issue(s) #2463  .

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/doi-job)

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/18f.gsa.gov/doi-job/)

[Preview README for this branch](https://github.com/18F/18f.gsa.gov/blob/doi-job/README.md)

Changes proposed in this pull request:
- Adds information about DOI's Office of Natural Resources Revenue to join page
- Adds information about ONRR Front End Engineer position and how to apply
- Created google form button 
- updated `open-positions.html` to hide 18F specific text 

/cc @rooneywp @gboone

### CODE REVIEW

File: _join/open-positions/onrr-front-end-engineer.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+ONRR is building an Innovation Design Team to help make this crucial data more open and useful, with the lofty goal of educating and informing public debate around natural resource management on public land.
+
+
+## Role summary:
+
+### Front End Engineer
+
+The Innovation Design Team will manage the U.S. Extractive Industries Transparency Initiative (USEITI) data portal and expand work begun in collaboration with 18F, as well as extending those efforts to other datasets and initiatives throughout ONRR. The Front End Engineer should be well-versed in modern development practices and data tools, utilizing: Sass, Jekyll, JavaScript and D3, SQLite, data APIs, and Node.js.
+
```
Comment (rooneywp) [inline _join/open-positions/onrr-front-end-engineer.md]: "Sass" should be "SAS" I believe.

File: _join/open-positions/onrr-front-end-engineer.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+- Participate in code review, architecture discussions, and feature prioritization.
+- Take pride of ownership in all projects you touch; leave code better than you found it. 
+- Contribute to documentation, tests, style fixes, accessibility, performance, security, etc.
+- Deliver projects that are easy to deploy, update, and monitor by ensuring the tooling for this is present early in the project development cycle.
+- Use usability research, analytics, and other metrics to influence project planning, design, and development.
+
+
+**Objective #2: You will find and create tools and resources to help ONRR address their data management needs.** 
+- Acquire broad knowledge of the landscape of past and present data management challenges that exist within ONRR. 
+- Recommend existing data-management tools and resources based on ONRR’s unique needs
```
Comment (rooneywp) [inline _join/open-positions/onrr-front-end-engineer.md]: Need a "." at the end of the line to maintain consistency. Same with the next point as well.

File: _join/open-positions/onrr-front-end-engineer.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- Prototype tools and/or build data management infrastructure for ONRR
+Conduct workshops to identify use cases and promote understanding of the data sets.  
+
+
+**Key Objective #3: You will act as an agent of culture change within ONRR**
+- Participate in or lead (as necessary) an open dialogue within ONRR on implementing modern development standards, including transparency, user-centered design, and agile methodologies. Understand and communicate the "why" of these standards, not just the "what."
```
Comment (rooneywp) [inline _join/open-positions/onrr-front-end-engineer.md]: Let's pull out the second sentence in this point, I don't think it's necessary.

File: _join/open-positions/onrr-front-end-engineer.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- Demonstrate resilience in the face of bureaucracy. When faced with a hurdle, determine the root cause and address that in addition to the hurdle itself.
+- Push the boundaries of what is possible within ONRR, while maintaining an understanding of the complexities of the organization. Push back against the things that can be pushed back against and take a longer view for the things that can't.
+- Own the vision and keep the team moving through technical leadership. Be an advocate for user-centered design practices.
```
Comment (rooneywp) [inline _join/open-positions/onrr-front-end-engineer.md]: Let's pull out the second sentence in this point, I don't think it's necessary.

File: _join/open-positions/onrr-front-end-engineer.md (hunk 1 above)
Comment (elainekamlley) [inline _join/open-positions/onrr-front-end-engineer.md]: Sass is correct. http://sass-lang.com/

**Pull request merged**